### PR TITLE
Packed project minor issues

### PR DESCRIPTION
**Pull request by glebmish**

### CODE REVIEW

File: src/appleseed/foundation/utility/unzipper.cpp [hunk 1]
```diff
@@ -175,7 +172,20 @@ void unzip(const string& zip_filename, const string& unzipped_dir)
     }
 }
 
-vector<string> get_filenames_with_extension_from_zip(const string& zip_filename, const string& extension)
+bool is_zip_file(const string& filename) {
```
Comment (dictoon) [inline src/appleseed/foundation/utility/unzipper.cpp]: Move brace to its own line.

File: src/appleseed/renderer/meta/tests/test_projectfilereader.cpp [hunk 2]
```diff
@@ -100,28 +100,30 @@ TEST_SUITE(Renderer_Modeling_Project_ProjectFileReader)
...
-
-            bf::remove_all(bf::path(project_unpacked));
-        }
-        catch (std::exception e)
-        {
-            bf::remove_all(bf::path(project_unpacked));
-            throw e;
-        }
-    }
+    // Test waits for a brilliant solution of how to invoke it without emitting error message
```
Comment (dictoon) [inline src/appleseed/renderer/meta/tests/test_projectfilereader.cpp]: :)

File: src/appleseed/foundation/utility/unzipper.h [hunk 3]
```diff
@@ -54,7 +54,7 @@ class UnzipException
 
 //
 // This function unzips zip file zipFilename
-// to unzipped_dir directory (filename.unzipped if NULL)
+// to unzipped_dir directory
 //
 // throws UnzipException in case of exception
```
Comment (dictoon) [inline src/appleseed/foundation/utility/unzipper.h]: Sentences should start with a capital and end with a period.

File: src/appleseed/renderer/modeling/project/projectfilereader.cpp [hunk 4]
```diff
@@ -3081,13 +3067,18 @@ auto_release_ptr<Project> ProjectFileReader::read(
...
+            bf::remove_all(bf::path(unpacked_project_directory));
+
         unzip(project_filepath, unpacked_project_directory);
         
+        const string project_name = appleseed_files[0];
         actual_project_filepath = (bf::path(unpacked_project_directory) / bf::path(project_name)).string().c_str();
         project_filepath = actual_project_filepath.data();
     }
 
     // Handle built-in projects.
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/project/projectfilereader.cpp]: I would move back handling of built-in projects at the beginning of the function, like it used to be. Just getting this special case out of the way before checking for packet project files, etc.

**Comment by dictoon** [comment]
Nice, ready to merge when Travis is done! Thank you.

**Comment by glebmish** [comment]
Great!

File: src/appleseed/foundation/utility/unzipper.cpp [hunk 5]
```diff
@@ -172,18 +172,17 @@ void unzip(const string& zip_filename, const string& unzipped_dir)
     }
 }
 
-bool is_zip_file(const string& filename)
+bool is_zip_file(const char* filename)
 {
-    try
+    unzFile zip_file = unzOpen(filename);
+
+    if (zip_file == NULL)
```
Comment (dictoon) [inline src/appleseed/foundation/utility/unzipper.cpp]: You should use `0` instead of `NULL` in C++03.

**Pull request merged**